opencti-platform/opencti-front/lang/front/fr.json

```
@@ -575,6 +576,7 @@
   "Create an artifact": "Créer un artefact",
   "Create an attack pattern": "Créer un motif d'attaque",
   "Create an attribute": "Créer un attribut",
+  "Create an dissemination list": "Créer une liste de diffusion",
```

```suggestion
  "Create a dissemination list": "Créer une liste de diffusion",
```
same everywhere with this phrase

---

Done